index.html

```diff
@@ -185,5 +185,37 @@ <h2>
         </div>
     </section>
 
+    <section class="contact-me">
+        <div class="form">
+            <h2>I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</h2>
+            <form action="https://formspree.io/f/mlezzzeo" method="post" class="form">
+                <input id="name" name="name" type="text" placeholder="Full Name" required="required">
```

Comment:
- [ ] for the name field there should be a maximum of 30 characters allowed.

Reply:
Fixed the maxlength attribute.